Makefile.am

```diff
@@ -1,3 +1,15 @@
 ACLOCAL_AMFLAGS = -I build/autotools
 DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --enable-introspection
 SUBDIRS=playerctl doc
+
+EXTRA_DIST = \
```

Comment:
I would advice you to get these two lines into master: *Still, sometimes there are files that must be distributed, but which are not covered in the automatic rules. These files should be listed in the EXTRA_DIST variable. You can mention files from subdirectories in EXTRA_DIST.*[source](https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html).